tests/tests_interface_pci.yml

```diff
@@ -0,0 +1,78 @@
+---
+# Tests interface_pci field, test must be used in VM
+- name: Test interfaces with PCI ids
+  hosts: all
+  become: true
+  roles:
+    - linux-system-roles.firewall
+
+  tasks:
+    - name: get backend from dbus
+      command: >-
+        dbus-send --system --print-reply --type=method_call
+        --dest=org.fedoraproject.FirewallD1
+        /org/fedoraproject/FirewallD1/config
+        org.freedesktop.DBus.Properties.Get
+        string:"org.fedoraproject.FirewallD1.config"
+        string:"FirewallBackend"
+      ignore_errors: yes
+      register: result
+
+    - name: get backend from result
+      set_fact:
+        nftables_backend: "{{ 'nftables' in result.stdout }}"
+
+    - name: test interfaces with PCI ids
+      block:
+        - name: add pci device ethernet controller
+          include_role:
+            name: linux-system-roles.firewall
+          vars:
+            firewall:
+              zone: internal
+              interface_pci_id: 1af4:0001
+              state: enabled
+              permanent: yes
+
+        - name: add pci device again
+          include_role:
+            name: linux-system-roles.firewall
+          vars:
+            firewall:
+              zone: internal
+              interface_pci_id: 1af4:0001
+              state: enabled
+              permanent: yes
+
+        - name: assert pcid not in nftable ruleset
+          command: nft list ruleset
+          register: result
+          failed_when: "result.rc != 0 or '1af4:0001' in result.stdout"
+          when: nftables_backend
```

Comment:
I think you need to use `nftables_backend | bool` here to avoid the "bare variable" problem